client/src/modules/stock/exit/exit.js

```diff
@@ -320,6 +320,10 @@ function StockExitController(
   function onLotSelect(row) {
     if (!row.lot || !row.lot.uuid) { return; }
 
+    if (row.lot.lifetime < 0) {
+      row.isExpired = true;
+    }
```

Comment:
I was surprised this was necessary.  Doesn't the server DB function Stock.lots.read() do that?  I think there is an 'expired' flag that is set on the lots in computeLotIndicators() that might be what you want.

Reply:
I'm still wondering if this is necessary.  Have you checked the value of the lot's '.expired' field.  When I look at line 770 of core.js, I see:

    lot.expired = !lot.exhausted && (lot.expiration_date < today);

Which seems very similar to what you need.

Reply:
I looked at the code again and would still like a response to my questions on this.   I see that this addition makes the status.tmp.html code a little cleaner -- but maybe not enough to be worth duplicating existing variables.